### PR TITLE
Update operators section in docs to include the latest helm-controller chart options

### LINKED ISSUE #260: Update operators section in docs to include the latest helm-controller chart options
The current docs are out of date, in which they still reference the helm-operator values. We now use a custom helm chart for helm-controller found at https://github.com/nitishm/charts/tree/main/helm-chart-sources/helm-controller

We need to update the docs with the values.yaml from the above source.

### PR DESCRIPTION
**Pull request by mahalrs**
Closes #260

### CODE REVIEW

**Comment by mahalrs** [comment]
@nitishm Let me know if I have correctly updated the docs.

**Comment by nitishm** [comment]
> @nitishm Let me know if I have correctly updated the docs.

Will take a look over the weekend! 👍

File: docs/operators.md [hunk 1]
```diff
@@ -366,251 +366,59 @@ chartmuseum:
...
 
-helm-operator:
-  # Default values for helm-operator
-  nameOverride: ""
-  fullnameOverride: ""
+helm-controller:
+  # Default values for helm-controller.
+  # This is a YAML-formatted file.
+  # Declare variables to be passed into your templates.
+  concurrent: 10
```
[the remark this answers is not in the document]
Comment (nitishm) [reply docs/operators.md:373]:
```suggestion
  concurrent: 5
```

**Review by nitishm — CHANGES_REQUESTED** [review]

**Review by nitishm — APPROVED** [review]
Awesome job @mahalrs !! 🚀 🎉

**Comment by jonathan-innis** [comment]
🚀  LGTM, congrats @mahalrs!

**Pull request merged**